ui/app/src/components/PendingApprovalDashlet/PendingApprovalDashlet.tsx

```diff
@@ -98,47 +104,69 @@ export function PendingApprovalDashlet(props: PendingApprovalDashletProps) {
   const selectedItems = Object.values(itemsById)?.filter((item) => selected[item.id]) ?? [];
   const selectionOptions = useSelectionOptions(Object.values(selectedItems), formatMessage, selectedCount);
   const site = useActiveSiteId();
+  const locale = useLocale();
   const { authoringBase } = useEnv();
   const dispatch = useDispatch();
+  const filterState = useDashletFilterState('pendingApprovalDashlet');
+  const refs = useUpdateRefs({
+    currentPage,
+    filterState
+  });
```

Comment:
Can we move `loadPagesUntil` here and keep a single _refs_? You could move this declaration below the others.